components/filters.tsx

```diff
@@ -199,6 +208,7 @@ export const Filters = () => {
           Private
         </button>
       </div>
+      &nbsp;
```

Comment:
sorry for confusion, Jasper meant to bring back the CSS space between buttons.  we actually had this before, we need to change back to the `space-x-2` style: https://github.com/nicolasdesmarais/ai-companion/commit/b83e2997811fc6b6e2ef5d88d2d67436ad253c49